wellcomeml/viz/visualize_clusters.py

```diff
@@ -0,0 +1,31 @@
+from bokeh.plotting import figure, show
+from wellcomeml.viz.palettes import Wellcome33
+from bokeh.io import output_notebook
+output_notebook()
```

Comment:
I'd actually not call `output_notebook` here, as users might actually want the `html` file instead.

Reply:
Changed to html